src/components/Theme/style.css

```diff
@@ -47,7 +47,7 @@
 }
 
 .mbottom-large {
-  margin-bottom: 4rem;
+  margin-bottom: 5.5rem;
```

Comment:
same thing here. if we are changing it, might as well not go with rem in this case. doesn't seem like the right approach.